frontend/src/pages/notebookImages/NotebookImages.tsx

```diff
@@ -23,8 +23,8 @@ const description = `Import, delete, and modify notebook images.`;
 const NotebookImages: React.FC = () => {
```

Comment:
Can you also rename the component, file name, and repo name to BYONImages? Also, could you update the other files in this repo?